.github/workflows/ci.yml

```
@@ -12,7 +12,7 @@ on:
 
 jobs:
   quality:
-    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@e35bf8e686fbbbd05ecc8c056ff3487b1b95f865
+    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@d6f1253ff160f53f17135ff21e12d760318c7d5c
```

```suggestion
    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@main
```

This should be main and always use the latest version, but I ran into some caching in actions where it was not using the latest version without forcing an updated commit hash. Any idea how we could get that to work?

---

Yes. Go back 🪖 to hard hat mode.

1. make releases here
2. use Dependabot where this workflow is used

---

BTW many actions do _moving tags_
e.g. `actions/checkout@v4`
https://github.com/actions/checkout/tags
`v4` is overwritten at each release to point to the latest one.

👴🏻 I am not so daring.